### PR TITLE
bugfix/header

### PR DESCRIPTION
**Pull request by DenisMuss**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-					<li class="contact"><a href="#"></a>Contact us</li>
-				</ul>
-			</nav>
-	</header>
-	
-
-</body>
+	<head>
+		<meta charset="UTF-8">
+		<title></title>
```
Comment (kkiirill) [inline index.html]: Add title for your website

File: index.html [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
+		<link rel="stylesheet" href="style.css">
+		<link href="https://fonts.googleapis.com/css2?family=Preahvihear&display=swap" rel="stylesheet">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+	</head>
+		<body>
```
Comment (kkiirill) [inline index.html]: body must be under head
<head>
</head>
<body>
</body>

File: index.html [hunk 3]
```diff
@@ -1,29 +1,25 @@
...
+			<header class="header">
+				<span class="logo">LOGO</span>
+					<nav class="nav">
```
Comment (kkiirill) [inline index.html]: span
under
nav

File: index.html [hunk 4]
```diff
@@ -1,29 +1,25 @@
...
+						<ul class="ul">
```
Comment (kkiirill) [inline index.html]: replace class ul to nav-list

File: index.html [hunk 5]
```diff
@@ -1,29 +1,25 @@
...
+							<li class="Home"><a href="#">Home</a></li>
```
Comment (kkiirill) [inline index.html]: what is class "Home"?

you need to add class to all link

name  class "nav-link"

File: style.css [hunk 6]
```diff
@@ -1,70 +1,63 @@
...
 
-.body {
-    background-color: black;
-}
-.a {
-text-decoration: none;
+:root {
+    --main-color: black;
+    --white-color: #ffffff;
+    --orange: #DC7000;
```
Comment (kkiirill) [inline style.css]: orange-color

Comment (kkiirill) [reply style.css]: write on lower case dc70000

File: style.css [hunk 7]
```diff
@@ -1,70 +1,63 @@
...
+    background-color: var(--main-color);
 }
 
-
 .header {
     display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 108px;
+    padding: 0%;
```
Comment (kkiirill) [inline style.css]: just 0

File: style.css [hunk 8]
```diff
@@ -1,70 +1,63 @@
...
+    margin: 0%;
```
Comment (kkiirill) [inline style.css]: just 0

File: style.css [hunk 9]
```diff
@@ -1,70 +1,63 @@
...
-    gap: 88px;
-    list-style: none; /* Убирает маркеры (если это список) */
-    padding: 0; /* Убирает внутренние отступы */
-    margin: 0; /* Убирает внешние отступы */
-    margin-top: 37px; 
-    margin-left: 689px; 
+    margin-right: 63px;
 }
 
-.nav ul li{
```
Comment (kkiirill) [inline style.css:32]: It's not a good practice

you need create class nav-link and remove this nav ul li 

read this https://developer.mozilla.org/ru/docs/Web/CSS/Specificity

File: style.css [hunk 10]
```diff
@@ -1,70 +1,63 @@
...
 }
 
-
-.layuot {
-max-width: 1300px;
-width: 100%;
-display: flex;
-align-items: center;
-justify-content: center;
+.ul li a {
```
Comment (kkiirill) [inline style.css]: not good practice use class

File: style.css [hunk 11]
```diff
@@ -1,70 +1,63 @@
...
+    color: var(--white-color);
+    text-decoration: none; 
 }
 
-.active {
-    border-bottom: 5px solid #DC7000;
+.Home {
```
Comment (kkiirill) [inline style.css]: you need to add this class not only for the first element. If you see multiple elements in a design and one element with an additional color/border etc., that means you need to add this functionality to all elements, but on hover

use hover 
read about it https://developer.mozilla.org/ru/docs/Web/CSS/Pseudo-classes
https://developer.mozilla.org/ru/docs/Learn/CSS/Building_blocks/Selectors/Pseudo-classes_and_pseudo-elements

File: style.css [hunk 12]
```diff
@@ -1,70 +1,63 @@
...
+    border-bottom: 4px solid var(--orange);
 }
 
 .contact {
     position: relative;
```
Comment (kkiirill) [inline style.css]: don't need this

File: style.css [hunk 13]
```diff
@@ -1,70 +1,63 @@
...
     font-family: 'Poppins', sans-serif;
     font-weight: 800;
     font-size: 29px;
-    display: flex;
-    width: 82px;
-    height: 45px;
-    margin-top: 34px; 
-    margin-left: 36px; 
-    color: #fff;
+    width: 82px;
```
Comment (kkiirill) [inline style.css]: Logo it's just text?
width and height doesn't need this

File: index.html (hunk 3 above)
Comment (DenisMuss) [reply index.html]: under???

File: style.css (hunk 8 above)
Comment (DenisMuss) [reply style.css]: corrected

File: style.css (hunk 10 above)
Comment (DenisMuss) [reply style.css]: fixed

Comment (DenisMuss) [reply style.css]: fixed

File: style.css (hunk 11 above)
Comment (DenisMuss) [reply style.css]: i used id on the link with a class

File: index.html [hunk 14]
```diff
@@ -1,25 +1,25 @@
...
+	<body>
+		<header class="header">
+			<span class="logo">LOGO</span>
+				<nav class="nav">
+					<ul class="ul-list">
+						<li class="nav-link" id="Home"><a href="#">Home</a></li>
+						<li class="nav-link"<a href="#">About us</a></li>
+						<li class="nav-link"><a href="#">Portfolio</a></li>
+						<li class="nav-link"><a href="#">News</a></li>
+						<li class="nav-link" id="Contact"><a href="#">Contact us</a></li>
```
Comment (kkiirill) [inline index.html]: why we need id=Contact?

File: style.css [hunk 15]
```diff
@@ -27,37 +27,34 @@ body {
     margin-right: 63px;
 }
 
-.ul {
+.ul-list {
```
Comment (kkiirill) [inline style.css]: it's not good practice naming ul li div etc - best use this naming nav-list

File: style.css [hunk 16]
```diff
@@ -27,37 +27,34 @@ body {
...
     display: flex;
+    text-align: center; 
+    line-height: 34px; 
     gap: 88px;
     list-style: none;
 }
 
-.ul li a {
+.nav-link a {
```
Comment (kkiirill) [inline style.css]: we don't need use "a" if we already use .nav-link

File: style.css [hunk 17]
```diff
@@ -27,37 +27,34 @@ body {
...
     color: var(--white-color);
     text-decoration: none; 
 }
 
-.Home {
-    border-bottom: 4px solid var(--orange);
+#Home {
```
Comment (kkiirill) [inline style.css]: why we need this?

File: style.css [hunk 18]
```diff
@@ -27,37 +27,34 @@ body {
...
+    border-bottom: 4px solid var(--orange-color);
 }
 
-.contact {
-    position: relative;
-    display: inline-block;
+#Contact {
```
Comment (kkiirill) [inline style.css]: not use id for this use class

**Review by kkiirill — APPROVED** [review]

**Review by kkiirill — CHANGES_REQUESTED** [review]
LGTM

**Review by kkiirill — APPROVED** [review]
Approve

**Pull request merged**